configure.ac

```diff
@@ -2358,8 +2359,16 @@ AC_ARG_WITH([libdpdk],
 )
 
 if test "x$with_libdpdk" != "xno"; then
+  PKG_CHECK_MODULES(DPDK, [libdpdk],
+    [LIBDPDK_LIBS="$DPDK_LIBS"],
```

Comment:
I haven't checked, but doesn't this override a custom LIBDPKD_LIBS passed in from the command line?